test/app-spec.js

```diff
@@ -0,0 +1,468 @@
+const request = require('supertest');
+var expect = require('chai').expect;
+var rewire = require('rewire');
+var app = rewire('../app');
+var meetup = rewire('../lib/meetups');
+
+describe("Questioner App",function(){
+	it("Loads the home page", function(done) {
+		request(app).get("/").expect(200).end(done);
+	});
+
+    describe("Get /api/meetup/:id",function(done){
+        it('returns a meetup',function(){
+            return request(app).get('/api/v1/meetups/1').
+                set('Accept','application/json').
+                expect(200,{
+                    "status": 200,
+                    "data": [
+                        {
+                            "tags": [
+                                "Events",
+                                "Meetsups",
+                                "Andela",

```

Comment:
Please use the specified 2 character spacing on this file.

Reply:
i have now used the 2 character spacing as advised i the code.